src/main/moditect/module-info.java

```diff
@@ -0,0 +1,9 @@
+module org.jsoup {
+    requires static java.xml;
+
+    exports org.jsoup;
```

Comment:
Don't we need to export .helper as well? I am concerned that tests didn't fail for this, are there tests to ensure we are exporting the correct packages?

Reply:
Actually, I'm not very clear with the package structure of Jsoup and not very sure who should be exported and who should not, so I just export `org.jsoup`. I'll add the necessary `exports` if you suggest some.